src/java.desktop/share/classes/javax/swing/JComponent.java

```diff
@@ -657,6 +657,7 @@ public JComponent() {
 
     /**
      * Resets the UI property to a value from the current look and feel.
+     * @implnote Since the default implementation of this method doesn't do anything,
```

Comment:
```suggestion
     * @implNote Since the default implementation of this method doesn't do anything,
```
The correct tag is `@implNote` with capital N.

Reply:
It was suggested to use @implnote tag and I saw that couple of places in jdk repo, we used that in Unsafe.java and TrayIcon.java so I used that, although in majority of places, @implNote is used, so I thought maybe both can be used at our discretion.
Anyway, will change to @implNote.

Reply:
I understand it. I just built the docs to see how it looks and the text after `@implnote` was missing — it's absolutely wrong this way. With `@implNote`, the text is displayed below *Implementation Note:* sub-heading.

Reply:
I can remove `Since` and have "@implNote The default implementation of this method doesn't do anything.
leaving other sentence `<code>JComponent</code> subclasses must override this method` separate. I guess that will keep the spec same as before and also keep the submitter satisfied.